version/7_1.sh

```diff
@@ -1,30 +1,31 @@
 #!/bin/bash -e
 
-PHP7_1_VERSION="7.1.24"
+PHP7_1_VERSION="7.1.25"
+PHPENV_VERSION="7.1"
```

Comment:
I think we should call this `PHPENV_VERSION_ALIAS` because that's what it really is. It's not actually the version of phpenv.